context/app/static/js/components/Detail/provenance/ProvVis/ProvData.js

```diff
@@ -46,6 +46,7 @@ export function makeCwlOutput(name, steps, extras) {
     meta: {
       global: true,
       in_path: true,
+      type: isReference ? 'reference file' : 'data file',
```

Comment:
Initially we opted to only provide a type for input files with `makeCwlInput`, but it's causing prop-types warnings. Is  it fine to add the same logic to output files or should this be some other value. We need something here.